ray_data_eval/video_inference/plotting/improve_tracing_view.py

```diff
@@ -0,0 +1,53 @@
+import os
```

Comment:
Do we need the ipynb file if we have this script?

Reply:
Yeah I think we can remove the notebook!